mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -100,10 +100,10 @@ def mask_cross_entropy(pred,
     """Calculate the CrossEntropy loss for masks.
 
     Args:
-        pred (torch.Tensor): The prediction with shape (N, C), C is the number
-            of classes.
+        pred (torch.Tensor): The prediction with shape (N, C, *), C is the
+            number of classes. The trailing * is spatio-temporal dimensions.
```

Comment:
The trailing * indicates arbitrary shape

Reply:
Fixed.